localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,344 @@
+ tsardom_solstice:0 "Tsardom"
+
+ ### Dryad group ###
+ # Feudal / Republic / Clan / Theocratic / Tribal
+ tender_male_dryad_group:0 "Tender"
+ tender_female_dryad_group:0 "Tender"
+ refuge_dryad_group:0 "Refuge"
+ guardian_male_dryad_group:0 "Guardian"
+ guardian_female_dryad_group:0 "Guardian"
+ grove_dryad_group:0 "Grove"
+ lord_male_dryad_group:0 "Lord"
+ lady_female_dryad_group:0 "Lady"
+ grand_grove_dryad_group:0 "Grand Grove"
+ high_lord_male_dryad_group:0 "High Lord"
+ high_lady_female_dryad_group:0 "High Lady"
+ domain_dryad_group:0 "Domain"
+ king_male_dryad_group:0 "King"
+ queen_female_dryad_group:0 "Queen"
+ kingdom_dryad_group:0 "Kingdom"
+
+ ### Tol'vir ###
+ # Feudal - Normal
+ governor_male_tolvir:0 "Governor"
+ governor_female_tolvir:0 "Governor"
+ region_tolvir:0 "Region"
+ nomarch_male_tolvir:0 "Nomarch"
+ nomarch_female_tolvir:0 "Nomarch"
+ nomes_tolvir:0 "Nomes"
+ pharaoh_male_tolvir:0 "Pharaoh"
+ pharaoh_female_tolvir:0 "Pharaoh"
+
+ # Feudal - Evil religions
+ dark_pharaoh_male_tolvir:0 "Dark Pharaoh"
+ dark_pharaoh_female_tolvir:0 "Dark Pharaoh"
+
+ ### Pandaren ###
+ # Feudal
+ leader_male_pandaren:0 "Leader"
+ leader_female_pandaren:0 "Leader"
+ district_pandaren:0 "District"
+ prefect_male_pandaren:0 "Prefect"
+ prefect_female_pandaren:0 "Prefect"
+ prefecture_pandaren:0 "Prefecture"
+ governor_male_pandaren:0 "Governor"
+ governor_female_pandaren:0 "Governor"
+ province_pandaren:0 "Province"
+
+ ### Mogu ###
+ # Feudal
+ leader_male_mogu:0 "Leader"
+ leader_female_mogu:0 "Leader"
+ district_mogu:0 "District"
+ lord_male_mogu:0 "Lord"
+ lady_female_mogu:0 "Lady"
+ commandery_mogu:0 "Commandery"
+ governor_male_mogu:0 "Governor"
+ governor_female_mogu:0 "Governor"
+ province_mogu:0 "Province"
+
+ ### Goblin ###
+ # Feudal / Republic
+ trade_lord_male_goblin:0 "Trade Lord"
+ trade_lady_female_goblin:0 "Trade Lady"
+ lordship_goblin:0 "Lordship"
+ trade_baron_male_goblin:0 "Trade Baron"
+ trade_baroness_female_goblin:0 "Trade Baroness"
+ high_barony_goblin:0 "High Barony"
+ trade_prince_male_goblin:0 "Trade Prince"
+ trade_princess_female_goblin:0 "Trade Princess"
+ trade_king_male_goblin:0 "Trade King"
+ trade_queen_female_goblin:0 "Trade Queen"
+ high_kingdom_goblin:0 "High Kingdom"
+
+ ### Centaur ###
+ # Feudal
+ khan_male_centaur:0 "Khan"
+ khan_female_centaur:0 "Khan"
+ khanat_centaur:0 "Khanat"
+ khagan_male_centaur:0 "Khagan"
+ khagan_female_centaur:0 "Khagan"
+ khaganat_centaur:0 "Khaganat"
+
+ ### Taurens ###
+ # Feudal/Clans/Tribal
+ chieftain_male_tauren:0 "Chieftain"
+ chieftain_female_tauren:0 "Chieftain"
+ high_chieftain_male_tauren:0 "High Chieftain"
+ high_chieftain_female_tauren:0 "High Chieftain"
+ great_chieftain_male_tauren:0 "Great Chieftain"
+ great_chieftain_female_tauren:0 "Great Chieftain"
+ patriarch_male_tauren:0 "Patriarch"
+ matriarch_female_tauren:0 "Matriarch"
 
- god_king_male:0 "God King"
- god_king_female:0 "God Queen"
\ No newline at end of file
+ ### Nerubian ###
+ # Feudal/Tribal
```

Comment:
Nerubian was led by King and Queen. Underking was sort of their Duchy level title (https://wowpedia.fandom.com/wiki/Azjol-Nerub_(kingdom)).